site/content/reference/openshift-permissions.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Openshift Permissions"
+date: 2024-05-14T15:08:41-07:00
+# Change draft status to false to publish doc
+draft: false
+# Description
+# Add a short description (150 chars) for the doc. Include keywords for SEO.
+# The description text appears in search results and at the top of the doc.
+description: "Learn about the permissions given to the NGINX Gateway Fabric Pod in an Openshift environment."
+# Assign weights in increments of 100
+weight: 100
+toc: true
+tags: [ "docs" ]
+# Create a new entry in the Jira DOCS Catalog and add the ticket ID (DOCS-<number>) below
+docs: "DOCS-000"
+---
+
+## Overview
+
+To deploy NGINX Gateway Fabric on an Openshift environment, additional permissions are granted to the Pod which are defined
+in a SecurityContextConstraints object. This document attempts to describe the permissions given to the Pod and
+why they were given.
+
+
+## Specification
+
+{{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
+| Name                       | Value                                                  | Explanation                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
Can I get some feedback on the explanations for the fields on the SecurityContextConstraints? I don't know how beneficial a lot of these statements basically saying "The NGF pod doesn't need it so the scc doesn't need it" is, and if we need an explanation for why the NGF pod doesn't need it.